static/app/views/routeNotFound.tsx

```diff
@@ -27,13 +29,14 @@ function RouteNotFound({router, location}: Props) {
       scope.setFingerprint(['RouteNotFound']);
```

Comment:
You could alternatively set the fingerprint here, though it'd be less flexible

Reply:
i think i might prefer to do it in settings!